src/views/DatasetManager/components/CreateDatasetButton/components/CreateDatasetWizard/CreateDatasetWizard.js

```diff
@@ -69,21 +69,11 @@ export const CreateDatasetWizard = ({ open, closeDialog }) => {
 
   const createDatasetAndCloseDialog = useCallback(() => {
     const values = methods.getValues();
-    const datasetToCreate = {
-      main: true,
-      name: values.name,
-      tags: values.tags,
-      description: values.description,
-      sourceType: values.sourceType,
-      source: {
-        location: values.location,
-        name: values.sourceName,
-        path: values.path,
-      },
-    };
-    createDataset(datasetToCreate);
+    if (datasetLocation === 'fromScratch') values.sourceType = 'None';
```

Comment:
Maybe create enum for dataSourceTypes ?

Reply:
:+1: for _sourceTypes_ but not sure about _datasetLocation_ because it's an internal state used only only for displaying purposes :slightly_smiling_face: